pyproject.toml

```diff
@@ -90,5 +98,45 @@ pretty = true
 show_error_codes = true
 strict = true
 
+[tool.pylint]
+reports=false
+py-version=3.6
```

Comment:
Shouldn't this be at least 3.8 since that's our min now (or maybe it's 3.7, can't remember...)?

Reply:
The CI flow is using 3.7.  I started this a long time ago, 
3.7 is not end fo life yet - I do not know what the policy is for codespell https://devguide.python.org/versions/ .

Reply:
Let's at least bump to 3.7 then

Reply:
Bumped.